orb.yml

```diff
@@ -51,14 +54,28 @@ jobs:
               reports:
                 - uri: << parameters.report_uri >> 
                   format: << parameters.report_format >>
-                  verbose: << parameters.report_verbosity >>" | tee ./repo/salus-configuration.yaml 
+                  verbose: << parameters.report_verbosity >>" | tee salus-configuration.yaml 
+          working_directory: /home
 
       - run: 
           name: Run scan
           command: |
-            SALUS_CONFIGURATION="file://salus-configuration.yaml" /usr/local/bundle/bin/bundle exec ./bin/salus scan
+            configuration_file="<<parameters.configuration_file>>"
+            if [[ -n "repo/${configuration_file}" ]]; then
+              if [[ ! -f "repo/${configuration_file}" ]]; then
+                echo "$repo/{configuration_file} does not exist, aborting."
```

Comment:
I like that we're being descriptive and aborting.

Reply:
Fixed a few typos here.